File: project_euler/p1.py
# To list the sum of all natural numbers below 1000 that are multiples of 3 or 5, 
sum = 0
for i in range(1000):
    if (i%3 == 0) or (i%5 == 0):
        sum = sum + i

print("The sum is ", sum)

File: project_euler/p1.cpp
#include<iostream>

using namespace std;

int main()
{
	int max, sum;
	sum = 0;
	max = 1000;
	for(int i=1;i<max;i++)
	{
		if((i%3 == 0) || (i%5 == 0))
		{
			sum = sum + i;
		}

	}

	cout<<"The sum is "<<sum<<endl;
	return 0;

}
